src/analog/CommandBinaryBuffer.ts

```diff
@@ -1477,6 +1507,42 @@ class CommandBinaryBuffer extends BinaryBuffer {
         this.setUint8(CommandBinaryBuffer.setMagneticInfluenceBit(byte, counter.isMagneticInfluence));
         this.setLegacyCounterValue(counter.value);
     }
+
+    getLegacyHourCounterWithDiff (): ILegacyHourCounterWithDiff {
+        const date = this.getDate();
+        const byte = this.getUint8();
```

Comment:
you don't have to create a var if you don't need it

```suggestion
        const isMagneticInfluence = CommandBinaryBuffer.getMagneticInfluenceBit(this.getUint8());
```

Reply:
it's needed for 2 usages: extract start hour from this byte + extract `isMagneticInfluence` bit